src-docs/src/views/notification_events/notification_events.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+
+import {
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+  EuiContextMenuItem,
+} from '../../../../src/components';
+
+import { EuiNotificationEventMeta } from '../../../../src/components/notification/notification_event_meta';
+
+export default () => {
+  const [isRead, setIsRead] = useState(false);
+
+  const onRead = () => {
+    setIsRead(true);
+  };
+
+  const panelStyle = { maxWidth: '400px' };
+
+  const contextMenuItems = [
+    <EuiContextMenuItem key="contextMenuItemA" onClick={onRead}>
+      Mark as read
+    </EuiContextMenuItem>,
+
+    <EuiContextMenuItem key="contextMenuItemB" onClick={() => {}}>
+      View messages like this
+    </EuiContextMenuItem>,
+
+    <EuiContextMenuItem key="contextMenuItemC" onClick={() => {}}>
+      Don’t notify me about this
+    </EuiContextMenuItem>,
+  ];
+
+  return (
+    <>
+      <EuiTitle size="s">
+        <h2>EuiNotificationEventMeta</h2>
+      </EuiTitle>
+      <EuiSpacer size="m" />
+      <EuiTitle size="xs">
+        <h3>All props</h3>
+      </EuiTitle>
+      <EuiSpacer size="s" />
+      <EuiPanel paddingSize="s" hasShadow={true} style={{ ...panelStyle }}>
+        <EuiNotificationEventMeta
+          type="Alert"
+          severity="Critical"
+          badgeColor="danger"
+          iconType="logoCloud"
+          time={<span>2 min ago</span>}
```

Comment:
`time` still accepts a `string` right? Might be good to have an example that was just the string version
```suggestion
          time="2 min ago
```